src/dfa.rs

```diff
@@ -202,6 +249,24 @@ impl<Symbol: Hash + Eq + Ord + Debug> Dfa<Symbol> {
 mod tests {
     use super::*;
 
+    /*
```

Comment:
don't have commented-out code. please add a valid test case for the Display trait implementation.